lib/node_modules/@stdlib/blas/ext/base/dnanasumors/lib/ndarray.js

```diff
@@ -52,20 +52,20 @@ function dnanasumors( N, x, stride, offset ) {
 	if ( N <= 0 ) {
 		return 0.0;
 	}
-	if ( N === 1 || stride === 0 ) {
-		if ( isnan( x[ offset ] ) ) {
+	if ( N === 1 || strideX === 0 ) {
+		if ( isnan( x[ offsetX ] ) ) {
 			return 0.0;
 		}
-		return abs( x[ offset ] );
+		return abs( x[ offsetX ] );
 	}
```

Comment:
I don't know why I originally included this conditional. It doesn't seem necessary (a) and (b) it does the incorrect thing when `strideX` is `0`. For example, I could have `N = 3` and `strideX = 0`. In which case, the result should be `abs(x[ix])*3`, not `abs(x[ix])`. I think this should be fixed, as I don't know/recall the original rationale.